modules/smart-agent_nagios-status-check/conf/readme.yaml

```diff
@@ -1,47 +1,28 @@
 documentations:
   - name: Smart Agent monitor
-    url: 'https://docs.signalfx.com/en/latest/integrations/agent/monitors/telegraf-exec.html'
+    url: 'https://docs.signalfx.com/en/latest/integrations/agent/monitors/nagios.html'
   - name: Nagios checks guidelines
     url: 'https://nagios-plugins.org/doc/guidelines.html#AEN78'
-  - name: Telegraf plugin exec
-    url: 'https://github.com/influxdata/telegraf/tree/master/plugins/inputs/exec'
-  - name: Telegraf parser nagios
-    url: 'https://github.com/influxdata/telegraf/tree/master/plugins/parsers/nagios'
 
 source_doc: |
-  There is no SignalFx official integration for `Nagios` but there is still a 
-  [monitor](https://docs.signalfx.com/en/latest/integrations/agent/monitors/telegraf-exec.html) to use.
 
   ### Monitors
 
-  You have to configure your existing Nagios script into the `telegraf/exec` monitor.
+  You have to configure your existing Nagios script into the `nagios` monitor.
   
   You can configure as many monitors as you have nagios checks to reuse but you have to define for each 
-  one at least:
 
-  * the `dataFormat: nagios` in `telegrafParser` parameter to parse status from script exit code.
-  * at least one dimension like `script` in `extraDimensions` to identify your check. If you do not 
-  define `aggregation_function` and let empty as by default it will work for all dimensions you choose.
-
-  Also it could be useful to adapt the `intervalSeconds` for each script.
+  it could be useful to adapt the `intervalSeconds` for each script.
 
   ### Examples
 
   ```yaml
-  - type: telegraf/exec
-    intervalSeconds: 180
-    command: /usr/local/bin/scripts/check-ipmitool.pl
-    extraDimensions:
-      script: check_ipmitool
-    telegrafParser:
-      dataFormat: nagios
-  - type: telegraf/exec
-    intervalSeconds: 900  
-    command: /usr/local/bin/scripts/check-megaraidsas
-    extraDimensions:
-      script: check_megaraidsas
-    telegrafParser:
-      dataFormat: nagios
+  - type: nagios
```

Comment:
may be can you add more examples, like different use cases

Reply:
I added another exemple to the doc